lru_cache/lru_cache.py

```diff
@@ -17,7 +19,10 @@ def __init__(self, limit=10):
     key-value pair doesn't exist in the cache.
     """
     def get(self, key):
-        pass
+        if self.storage.get(key, False):
```

Comment:
Nice use of the `get()` with the optional second argument if the key does not exist